server/emailsending.js

```diff
@@ -1,47 +0,0 @@
-const express = require("express");
```

Comment:
this PR is removing emailsending.js - seems to me quite important. This is just an footer styling PR so shouldn't be commiting any other changes.